### PR TITLE
sourcery refactor

### PR DESCRIPTION
**Pull request by ervinpopescu**
Using `sourcery review --fix libqtile/`.

### CODE REVIEW

**Comment by tych0** [comment]
I don't like patches like this for a number of reasons, but if we're even going to consider merging it, the tool needs to be in CI so we don't violate its requirements in the future and have to do another big sweeping change like this.

**Comment by elParaguayo** [comment]
@tych0 I think this was done out of interest and they submitted a PR to share it. Agreed that this won't get merged but it's interesting to see the changes it made.

**Comment by ervinpopescu** [comment]
Yep, @elParaguayo is right. I just wanted to see if it would pass the tests :)

**Comment by jwijenbergh** [comment]
Indeed way too many changes to review, but thanks for the pr. Definitely looks like a neat tool

File: libqtile/backend/x11/core.py [hunk 1]
```diff
@@ -315,13 +314,6 @@ def _xpoll(self) -> None:
                     if not ret:
                         break
 
-            # Catch some bad X exceptions. Since X is event based, race
```
Comment (jwijenbergh) [inline libqtile/backend/x11/core.py:318]: wait what :D

Comment (ervinpopescu) [reply libqtile/backend/x11/core.py:318]: yeah don't mind it :) it's just spitting out garbage sometimes, I'm gonna take a look at what it did the following days.

Comment (ervinpopescu) [reply libqtile/backend/x11/core.py:318]: You were talking about the block comment tho or what? cause I can't figure it out.

Comment (jwijenbergh) [reply libqtile/backend/x11/core.py:318]: Yeah the fact that it seems to remove comments. Also when it simplifies code to a one-liner the comments are gone :(

Comment (ervinpopescu) [reply libqtile/backend/x11/core.py:318]: Yeah, @elParaguayo saw that too. I'll look into it.

Comment (jwijenbergh) [reply libqtile/backend/x11/core.py:318]: Thanks for doing this :)

**Comment by ervinpopescu** [comment]
```console
$ git diff sourcery-refactor~~ | grep -E '\-[[:space:]]+#' | wc -l
92
```
Could be worse :)

File: libqtile/command/client.py [hunk 2]
```diff
@@ -294,17 +293,18 @@ def normalize_item(self, item: str) -> str | int:
...
-                str(object_type),
-                [(str(object_type), str(item))],
-            )
-    else:
+    if object_type in ["group", "widget", "bar"] or object_type not in [
+        "layout",
+        "window",
+        "screen",
+    ]:
         return item
```
Comment (ervinpopescu) [inline libqtile/command/client.py:301]: this is funny xD

**Comment by ervinpopescu** [comment]
Well, I've decided that's all the time I had available for this. Thanks for chiming in :)

**Pull request closed without merging**